service/src/main/resources/api/service_openapi.yaml

```diff
@@ -318,6 +318,9 @@ components:
         createdDate:
           type: string
           description: Date the profile was created
+        lastModified:
+          type: string
+          description: Date the profile was last modified
```

Comment:
The documentation here isn't quite correct, this is the timestamp in UTC the profile was last modifed, not the date.

Reply:
@yunip20 can you update this documentation as well?